e2e/cli-e2e/tests/print-config.spec.ts

```diff
@@ -69,5 +68,17 @@ describe('print-config', () => {
       plugins: expect.any(Array),
       categories: [],
     });
+    expect(args.persist.format).toEqual(undefined);
```

Comment:
The `.toEqual()` above already checks this (extra properties would cause it to fail).

```suggestion
```